lix/README.md

```diff
@@ -1,11 +1,10 @@
 # Lix (previously Project Lisa)
 
-Lix is the git compatible data layer for inlang apps.
+Lix is the git compatible data storage for applications with an "edit > review > automate" workflow.
```

Comment:
We should refer to lix as change control system :D 

> Lix is a change control system for applications.

I think that's sufficient: 

- "what is change control?" -> we will form the term over time
- "aha, i can build apps with it" -> key message

Reply:
git compatible is probably also not important: 

- we don't want to raise the thought that lix is like git
- i don't expect devs to adopt lix because of git compatibility

if the assumptions above hold untrue, we can update the readme.